addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -763,6 +763,8 @@ QUnit.test('chat window: state conservation on toggle home menu', async function
     // Show home menu
     await this.env.bus.trigger('will_show_home_menu');
     await this.env.bus.trigger('show_home_menu');
+    // in real condition, chat window will be unmount and recreated
+    await this.env.services['chat_window']._mount();
```

Comment:
Maybe we would want to do this with a hint instead of always at mount? Not really sure though.

Reply:
I don't know but in tests by the way if we want to totally mock a home menu show, we should unmount and remount